ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -75,6 +78,9 @@
   private final KsqlRestClient restClient;
   private final Console terminal;
 
+  private long lastCommandSequenceNumber;
+  private boolean shouldWaitForPreviousCommand;
```

Comment:
as per previous comment, consider
```suggestion
  private boolean requestPipelining;
```

(Though note that this is the opposite of the current name, i.e. we should wait for previous command to finish if `requestPipelining` is _false_.)